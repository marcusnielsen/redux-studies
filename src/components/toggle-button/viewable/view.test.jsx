import {shallow} from 'enzyme'
import React from 'react'
import PropTypes from './prop-types'
import View from './view'

test('view', function onTest () {
  const propTypes = PropTypes({reactPropTypes: React.PropTypes})
  const ReactView = View({React, propTypes})
  const wrapper = shallow(<ReactView active={false} />)
  expect(wrapper.html()).toMatchSnapshot()
})
