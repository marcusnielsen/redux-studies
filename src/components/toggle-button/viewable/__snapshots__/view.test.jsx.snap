exports[`test view 1`] = `"<div>false</div>"`;
